lxd/daemon.go

```diff
@@ -369,6 +369,16 @@ func (d *Daemon) Authenticate(w http.ResponseWriter, r *http.Request) (bool, str
 		}
 	}
 
+	// Validate metrics certificates.
+	if shared.StringInSlice(r.URL.Path, []string{"/1.0", "/1.0/metrics"}) {
```

Comment:
I think we should be able to just do `/1.0/metrics` here. Both `/` and `/1.0` are accessible unauthenticated so shouldn't need special casing.